src/item.ts

```diff
@@ -67,6 +70,11 @@ type Video = {
     width: string;
 }
 
+type ContentAtom = {
+    kind: ElementKind.ContentAtom;
+    atom: Atom;
```

Comment:
How much information do we need from this type? Could we create a narrower type that looks something like this:

```ts
const enum AtomKind {
    Interactive,
    Recipe,
}

interface Atom {
    kind: AtomKind;
    js: string;
    css: string;
    html: string;
}
```